developer-guides/grpc-web-ui.md

```diff
@@ -0,0 +1,67 @@
+---
+description: >-
+  A general introduction to gRPC Web UI.
+cover: ../.gitbook/assets/Discord Invite (29).png
+coverY: 258
+---
+
+## Introduction
+
+## gRPC Web UI - BETA <a href="gRPC-Web-Ui" id="grpc-web-ui"></a>
+
+The **gRPC Web UI** is a tool for querying gRPC. Currently in **BETA**, this web-based user interface allows you to interact with gRPC servers directly from browser. Similar to Postman, but specifically designed for gRPC APIs instead of REST.
+
+This tool can be considered an extension to [grpcurl](https://github.com/fullstorydev/grpcurl). While `grpcurl` is a command-line interface, `grpcui` provides a web/browser-based GUI, allowing you to interactively construct requests to send to a gRPC server.
+
+With this tool, you can also browse the schema for gRPC services, presented as a list of available endpoints. This is enabled either by querying a server that supports [server reflection](https://github.com/grpc/grpc/blob/master/src/proto/grpc/reflection/v1/reflection.proto), by reading proto source files, or by loading compiled "protoset" files (files that contain encoded file [descriptor protos](https://github.com/google/protobuf/blob/master/src/google/protobuf/descriptor.proto)). The tool uses this schema to transform JSON request data into binary encoded protobufs. Therefore, if the server you interact with does not support reflection, you will need the proto source files that define the service or the protoset files that `grpcui` can use.
```

Comment:
This is implementation detail for grpc, I think its no necessary here since the document is targeted to devs on juno. Its sufficient to mention what grpc UI brings to juno devs.